python/ray/experimental/packaging/load_package.py

```diff
@@ -56,14 +70,25 @@ def load_package(config_path: str) -> "_RuntimePackage":
         >>> def f(): ...
     """
 
-    config_path = _download_from_github_if_needed(config_path)
+    if isinstance(package_def, dict):
+        # TODO(edoakes): when loading the package natively in Python, we should
+        # support passing in the stub module directly instead of a file.
+        config = package_def
+        base_dir = os.path.abspath(os.path.dirname(__file__))
```

Comment:
Shouldn't this be relative to the stub file not load_package.py?

Reply:
Actually maybe it doesn't make sense to infer base_dir here, so we should set it to None and handle appropriately below.